lib/value_range_check/value_range_check_base.jinja

```diff
@@ -33,9 +33,11 @@
    when mould is larger than value.) #}
 
 {% if ALL_TYPES is not defined %}
-   {% set ALL_TYPES = [ ("Double", "real(kind=real64)",   64),
-                        ("Real",   "real(kind=real32)",   32),
-                        ("Int",    "integer(kind=int32)", 32) ] %}
+   {% set ALL_TYPES = [ ("Double",  "real(kind=real64)",   64),
+                        ("Real",    "real(kind=real32)",   32),
+                        ("Char",    "character",            8),
+                        ("Logical", "logical",              8),
```

Comment:
I happened to be reading about this today and, in Fortran, the storage of a logical has to be the same as the storage of a default integer. I don't know what it is for character. I can't remember how this 'number of bits' is used here though?

Reply:
OK, that cost me some time. Turns out, the bit size was only used for the read-only-verification, so I decided to remove these unnecessary value from the other files.
But since it was (potentially/likely) also set when calling `process.py`, so I changed read-only to  set the bit size depending on name instead. That made this patch a bit bigger, but these unnecessary values are gone now.

Reply:
Sorry. That was quite a lot of work. I think it's an improvement though.